### PR TITLE
Adding page for invalid invitation links

### LINKED ISSUE #94: Error when trying to accept a team invitation link with the wrong user
## Error

```
Error: Request failed with status code 406
    at createError (/Users/bandesz/src/github.com/appvia/kore-ui/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/Users/bandesz/src/github.com/appvia/kore-ui/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/Users/bandesz/src/github.com/appvia/kore-ui/node_modules/axios/lib/adapters/http.js:237:11)
    at IncomingMessage.emit (events.js:328:22)
    at endReadableNT (_stream_readable.js:1201:12)
    at processTicksAndRejections (internal/process/task_queues.js:84:21)
```

## Proposed solution

We should show an error on the UI saying that the invitation link is invalid.

### PR DESCRIPTION
**Pull request by davet1985**
Screen Shot 2020-03-19 at 10 37 42

Fixes #94

### CODE REVIEW

File: pages/invalid-team-invitation.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  const link = router.query.link
+
+  return (
+    <div>
+      <Result
+        status="error"
+        title="Invalid invitation link"
+        subTitle={
+          <div>
+            <Paragraph>The link you used: <Text style={{ overflowWrap: 'break-word' }}><a href={link}>{link}</a></Text></Paragraph>
```
Comment (mrsheepuk) [inline pages/invalid-team-invitation.js]: perhaps add " is not valid" somewhere here? I might be missing hte wider context of where this is displayed though.

Comment (davet1985) [reply pages/invalid-team-invitation.js]: I forgot to add the screenshot 😱

Comment (davet1985) [reply pages/invalid-team-invitation.js]: @mrsheepuk I've added a screenshot, let me know what you think.

**Comment by mrsheepuk** [comment]
Clear now with the screenshot, lgtm.

**Pull request merged**